pkg/test/echo/docker/Dockerfile.app

```diff
@@ -1,5 +1,4 @@
-# hadolint ignore=DL3006
-FROM istionightly/base_debug
+FROM ubuntu:xenial
```

Comment:
nit - any reason to change this?

Reply:
will revert.

Reply:
oh yes, the reason to change this is have independent base image updates.  Unfortunately, I wasn't able to get the PRE logic to work in this particular dockerfile because it has some intermediate bash.

Reply:
ahh this breaks tests because they require curl

Reply:
i reverted this change as it doesn't affect the implementation runtime.  This does need attention later after 1.3 is pinned.